spicy/toolchain/src/compiler/codegen/productions/switch.cc

```diff
@@ -26,7 +26,7 @@ std::string codegen::production::Switch::dump() const {
 
         exprs.reserve(c.first.size());
         for ( const auto& e : c.first )
-            exprs.push_back(hilti::util::fmt("%s", e));
+            exprs.push_back(hilti::util::fmt("%s", e->print()));
```

Comment:
Good idea, I like dereferencing better since that way we do no explicitly construct a `string` (might still happen in the formatting library, but up to them). It also is shorter 😎.